index.html

```diff
@@ -0,0 +1,133 @@
+        </p>
+      </section>
+
+      <section class="content-box" id="online-reputation-management">
+        <img
+          src="./assets/images/online-reputation-management.jpg"
+          class="float-right"
+          alt="a businessman on his phone, with a laptop portraying the label 'Reputation' alongside some graphs"
+        />
+
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </section>
+
+      <section class="content-box" id="social-media-marketing">
+        <img
+          src="./assets/images/social-media-marketing.jpg"
+          class="float-left"
+          alt="employees around a desk with laptops and animations of social media icons"
+        />
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </section>
+    </main>
+
+    <!-- Aside section containing icons and brief pieces of text -->
+    <aside class="benefits">
+      <section class="benefit-subsection">
+        <h3>Lead Generation</h3>
+        <img
+          src="./assets/images/lead-generation.png"
+          alt="a cogwheel pushing down into a funnel with a dollar sign"
+        />
+        <p>
+          Inbound strategies for lead generation require less work for your
+          business, bringing customers directly to your website.
+        </p>
+      </section>
+      <section class="benefit-subsection">
+        <h3>Brand Awareness</h3>
+        <img
+          src="./assets/images/brand-awareness.png"
+          alt="an icon of a person in a suit with a lightbulb for a head"
+        />
+        <p>
+          Users find your business through paid and organic searches, increasing
+          the search ranking and visibility for your business.
+        </p>
+      </section>
+      <section class="benefit-subsection">
+        <h3>Cost Management</h3>
+        <img
+          src="./assets/images/cost-management.png"
+          alt="a symbol of cogwheels and money symbols"
+        />
+        <p>
+          As the search ranking for your business increases, your advertising
+          costs decrease, and you no longer need to advertise your page.
+        </p>
+      </section>
+    </aside>
+
+    <!-- Footer section containing company information -->
+    <footer class="footer-wrapper">
+      <h4>Made with ❤️️ by Horiseon</h4>
```

Comment:
As this is a footer, this doesn't need to be a `h4` as it's in a different section, but I like that you followed the correct structure for your `h` tags.

Reply:
Understood!